WalletWasabi.Fluent/Views/Wallets/Send/PrivacyWarningsView.axaml

```diff
@@ -143,7 +146,8 @@
           <Border Classes="warning info">
             <DockPanel>
               <PathIcon />
-              <TextBlock Text="Transaction creates change." />
+              <TextBlock Text="Transaction creates change." 
+                         ToolTip.Tip="Change can be used to link this transaction with the next one that will use it. If possible use the change avoidance suggestion, or coinjoin the change later." />
```

Comment:
```suggestion
                         ToolTip.Tip="Change can be used to link this transaction with the next one that will use it. Send a little more or less if this is ok for the receiver, or coinjoin the change later." />
```

Reply:
Suggesting to use change avoidance is more precise here because saying `Send a little more or less if this is ok for the receiver` might lead users to manually edit the amount.

Maybe we can combine both: `Use the change avoidance suggestion to send a little more or less if this is ok for the receiver`